<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>713</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">backgorund.image:
url(:/Imagenes Pro/mlyr180400066.jpg)
background-color: rgb(211, 215, 207);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="enviar">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Enviar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="carnet">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_valores">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>351</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 9pt &quot;Ubuntu&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Bitstream Charter&quot;;
background-color: rgb(255, 255, 255);
font: 20pt &quot;Noto Mono&quot;;
color: rgb(239, 41, 41);</string>
    </property>
    <property name="text">
     <string>ROJO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Bitstream Charter&quot;;
background-color: rgb(255, 255, 255);
color: rgb(115, 210, 22);
font: 20pt &quot;Noto Mono&quot;;</string>
    </property>
    <property name="text">
     <string>VERDE</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>721</width>
      <height>711</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Imagenes Pro/mlyr180400066.jpg);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="val_tem">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>125</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="val_vel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>350</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="val_vol">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>350</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="val_frec">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>350</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_operation_time">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>510</y>
      <width>331</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Ubuntu&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Valores">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>480</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;VALORES&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;Sans&quot;;
color: rgb(32, 74, 135);</string>
    </property>
    <property name="text">
     <string>VALORES</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="operacion">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>480</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;Sans&quot;;
color: rgb(52, 101, 164);</string>
    </property>
    <property name="text">
     <string>TIEMPO DE OPERACION</string>
    </property>
   </widget>
   <widget class="QLabel" name="fecha">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>380</y>
      <width>481</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 25pt &quot;Noto Mono&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="texto">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Bitstream Charter&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Registro de datos:</string>
    </property>
   </widget>
   <widget class="QLabel" name="texto_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Bitstream Charter&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Tiempo de Inicio:</string>
    </property>
   </widget>
   <widget class="QLabel" name="texto_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>320</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Bitstream Charter&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Temperatura</string>
    </property>
   </widget>
   <widget class="QLabel" name="texto_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>320</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Bitstream Charter&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Voltaje</string>
    </property>
   </widget>
   <widget class="QLabel" name="texto_5">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>320</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Bitstream Charter&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Frecuencia</string>
    </property>
   </widget>
   <widget class="QLabel" name="texto_6">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>320</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Bitstream Charter&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Velocidad</string>
    </property>
   </widget>
   <widget class="QLabel" name="texto_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Bitstream Charter&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Reporte:</string>
    </property>
   </widget>
   <widget class="QLabel" name="date_time">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Sans&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="texto_8">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Bitstream Charter&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Indica cuando el valor es optimo para el motor</string>
    </property>
   </widget>
   <widget class="QLabel" name="texto_9">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Bitstream Charter&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Indica cuando el valor no es optimo para el motor</string>
    </property>
   </widget>
   <widget class="QLabel" name="texto_10">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Bitstream Charter&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Digite su N° de Carnet para validar los datos</string>
    </property>
   </widget>
   <widget class="QLabel" name="texto_11">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>471</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;Bitstream Charter&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Control de Motor A.C.</string>
    </property>
   </widget>
   <zorder>textEdit</zorder>
   <zorder>enviar</zorder>
   <zorder>carnet</zorder>
   <zorder>table_valores</zorder>
   <zorder>label_21</zorder>
   <zorder>label_23</zorder>
   <zorder>val_tem</zorder>
   <zorder>val_vel</zorder>
   <zorder>val_vol</zorder>
   <zorder>val_frec</zorder>
   <zorder>table_operation_time</zorder>
   <zorder>Valores</zorder>
   <zorder>operacion</zorder>
   <zorder>fecha</zorder>
   <zorder>texto</zorder>
   <zorder>texto_2</zorder>
   <zorder>texto_3</zorder>
   <zorder>texto_4</zorder>
   <zorder>texto_5</zorder>
   <zorder>texto_6</zorder>
   <zorder>texto_7</zorder>
   <zorder>date_time</zorder>
   <zorder>texto_8</zorder>
   <zorder>texto_9</zorder>
   <zorder>texto_10</zorder>
   <zorder>texto_11</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
